src/driver/sqlite/SqliteQueryRunner.ts

```diff
@@ -68,14 +68,9 @@ export class SqliteQueryRunner extends AbstractSqliteQueryRunner {
             const queryStartTime = +new Date()
             const isInsertQuery = query.startsWith("INSERT ")
             const isDeleteQuery = query.startsWith("DELETE ")
-            const isUpdateQuery = query.startsWith("UPDATE ")
 
             const execute = async () => {
-                if (isInsertQuery || isDeleteQuery || isUpdateQuery) {
-                    await databaseConnection.run(query, parameters, handler)
-                } else {
-                    await databaseConnection.all(query, parameters, handler)
-                }
+                await databaseConnection.all(query, parameters, handler)
```

Comment:
also please explain this change, not clear why it was needed.

Reply:
These lines 69-71 check for `query.startsWith("INSERT ")`... But this is actually bad practice. It's not even parsed!
A counterexample is the 'WITH' clause: [https://www.sqlite.org/lang_with.html](https://www.sqlite.org/lang_with.html)

Anyway, this would probably be OK if this whole .query has an internal invocation that can do this and the external RAW uses ONLY sqlite3.all and no conditional logic...